feat: add apache example to ubuntu installer config (#282)
canonical/landscape-documentation: docs/how-to-guides/ubuntu-installer/configure-landscape-deployment-for-autoinstall-provisioning.md

The Ubuntu installer (26.04 and later) can use Landscape to serve an autoinstall file. Your Landscape account must use OIDC authentication.

```{note}

> This feature is available from Landscape server `25.10` onwards, and only on self-hosted deployments. It is not intended for {ref}`Quickstart <how-to-quickstart-installation>` deployments.

Reviewer: ```suggestion
This feature is available from Landscape server `25.10` onwards, and only on self-hosted deployments. {ref}`Quickstart <how-to-quickstart-installation>` deployments don't support using autoinstall.
```
If that's true. If it does technically support it but it's just not recommended, then leave as you originally had it.

Author: it does support it, just not recommended

- The installer connects using HTTPS.
- The `landscape-ubuntu-installer-attach` service listens on port 53354 by default.

> #### Example configuration

Reviewer: I'd suggest having a clearer break, so it's more obvious that there are two different examples here.

#### Example configuration

**HAProxy**

[If you're using HAProxy, add the following..... (etc)]

**Apache**

[If you're using Apache, add the following...... (etc)]

Reviewer: you can keep the H5 heading for Example Config (it didn't show in my comment, just looks bolded)
